pkgs/desktops/gnome-3/core/tracker-miners/default.nix

```diff
@@ -24,18 +20,55 @@ in stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  nativeBuildInputs = [ vala pkgconfig intltool itstool libxslt wrapGAppsHook ];
-  # TODO: add libgrss, libenca, libosinfo
-  buildInputs = [
-    bzip2 evolution-data-server exempi flac giflib glib gnome3.totem-pl-parser
-    gnome3.tracker gst_all_1.gst-plugins-base gst_all_1.gstreamer icu
-    json-glib libcue libexif libgsf libiptcdata libjpeg libpng libseccomp libsoup
-    libtiff libuuid libvorbis libxml2 poppler taglib upower gexiv2
+  mesonFlags = [
+    "-Ddbus_services=share/dbus-1/services"
```

Comment:
This seems to produce dangling symlinks in `share/tracker/miners/`. We really need https://github.com/NixOS/nixpkgs/pull/37693 and `${placeholder "out"}`. For now, we could use the horrendous

```nix
  preConfigure = ''
    mesonFlagsArray+("-Ddbus_services=$out/share/dbus-1/services")
  '';
```